apps/core/src/dashboards/dashboards.e2e.spec.ts

```diff
@@ -1284,6 +1284,116 @@ describe('DashboardsModule', () => {
     });
   });
 
+  describe('DELETE /dashboards/bulk HTTP/1.1', () => {
+    test('returns 200 on success', async () => {
+      const dashboard1 = await seedTestDashboard('dashboard 1');
+      const dashboard2 = await seedTestDashboard('dashboard 2');
+      const dashboard3 = await seedTestDashboard('dashboard 3');
+
+      const deleteResponse = await app.inject({
+        headers: {
+          Authorization: `Bearer ${bearerToken}`,
+        },
+        method: 'DELETE',
+        payload: {
+          ids: [dashboard1.id, dashboard2.id, dashboard3.id],
+        },
+        url: `/dashboards/bulk`,
+      });
+
+      expect(deleteResponse.statusCode).toBe(200);
+      expect(JSON.parse(deleteResponse.body)).toEqual({
+        deletedIds: [dashboard1.id, dashboard2.id, dashboard3.id],
+      });
+
+      const getResponse1 = await app.inject({
+        headers: {
+          Authorization: `Bearer ${bearerToken}`,
+        },
+        method: 'GET',
+        url: `/dashboards/${dashboard1.id}`,
+      });
+
+      const getResponse2 = await app.inject({
+        headers: {
+          Authorization: `Bearer ${bearerToken}`,
+        },
+        method: 'GET',
+        url: `/dashboards/${dashboard2.id}`,
+      });
+
+      const getResponse3 = await app.inject({
+        headers: {
+          Authorization: `Bearer ${bearerToken}`,
+        },
+        method: 'GET',
+        url: `/dashboards/${dashboard3.id}`,
+      });
+
+      expect(getResponse1.statusCode).toBe(404);
+      expect(getResponse2.statusCode).toBe(404);
+      expect(getResponse3.statusCode).toBe(404);
+    });
+
+    test('returns 200 when called with empty id list', async () => {
```

Comment:
Can you add one more test for 'returns 200 when called with a single item list'? I think that can sometimes be an edge case when moving from single-item to multi-item data.

Reply:
Yup, will do.